cmd/podman/build.go

```diff
@@ -269,6 +296,7 @@ func buildCmd(c *cliconfig.BuildValues) error {
 		SignaturePolicyPath:     c.SignaturePolicy,
 		Squash:                  c.Squash,
 		Target:                  c.Target,
+		NamespaceOptions:        nsValues,
```

Comment:
nit of a nit, the other options here are ordered alphabetically, could you move this one up please?